lib/pure/os.nim

```diff
@@ -1031,7 +1031,10 @@ type
     blockSize*: int                   ## Preferred I/O block size for this object.
                                       ## In some filesystems, this may vary from file to file.
     isRegular*: bool                  ## Is file regular? (on Unix some "files"
-                                      ## can be non-regular like FIFOs, devices)
```

Comment:
Maybe we should flip this around and call it `isFragile` then a default value of `false` makes sense.